File: Examples/VisualBasic/Programming-Documents/Document/ComboBoxContentControl.vb
Imports System.IO
Imports Aspose.Words
Imports Aspose.Words.Markup
Imports System.Drawing

Class ComboBoxContentControl
    Public Shared Sub Run()
        ' ExStart:ComboBoxContentControl
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithDocument()
        Dim doc As New Document()
        Dim sdt As New StructuredDocumentTag(doc, SdtType.ComboBox, MarkupLevel.Block)

        sdt.ListItems.Add(New SdtListItem("Choose an item", "-1"))
        sdt.ListItems.Add(New SdtListItem("Item 1", "1"))
        sdt.ListItems.Add(New SdtListItem("Item 2", "2"))
        doc.FirstSection.Body.AppendChild(sdt)

        dataDir = dataDir & Convert.ToString("ComboBoxContentControl_out_.docx")
        doc.Save(dataDir)
        ' ExEnd:ComboBoxContentControl
        Console.WriteLine(Convert.ToString(vbLf & "Combo box type content control created successfully." & vbLf & "File saved at ") & dataDir)
    End Sub
End Class

File: Examples/CSharp/Programming-Documents/Document/ComboBoxContentControl.cs

using System.IO;
using Aspose.Words;
using System;
using Aspose.Words.Markup;
using System.Drawing;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_With_Document
{
    class ComboBoxContentControl
    {
        public static void Run()
        {
            //ExStart:ComboBoxContentControl
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithDocument();
            Document doc = new Document();
            StructuredDocumentTag sdt = new StructuredDocumentTag(doc, SdtType.ComboBox, MarkupLevel.Block);

            sdt.ListItems.Add(new SdtListItem("Choose an item", "-1"));
            sdt.ListItems.Add(new SdtListItem("Item 1", "1"));
            sdt.ListItems.Add(new SdtListItem("Item 2", "2"));
            doc.FirstSection.Body.AppendChild(sdt);

            dataDir = dataDir + "ComboBoxContentControl_out_.docx";
            doc.Save(dataDir);
            //ExEnd:ComboBoxContentControl
            Console.WriteLine("\nCombo box type content control created successfully.\nFile saved at " + dataDir);
        }        
    }
}
